### DISCUSSION TITLE
InfluxDB

### DISCUSSION THREAD

**Discussion by PatrickBinzer** [Budibase/budibase#1898] [Ideas]
Is it possible to get an integration for influxdb as external datasource?

**Comment by mjashanks** [comment]
Hey @PatrickBinzer. It's not going to be done on our immediate roadmap. We are always open to contributions, if you are able to write it yourself in NodeJS?

I've tagged it under integrations - if it gets some good upvotes, we'll definitely look at it. Or, if you are running a large deployment of Budibase (or intend to), there is potential for us to prioritise it.
